### ISSUE TITLE
H&L: FishSamplingEntryDialog: Consider modifying HookMatrix tags to include Set ID + ADH and not just ADH, for one could scan an ADH from a previous site accidentally

### ISSUE THREAD

**Issue by wsmith-nwfsc** [nwfsc-fram/pyFieldSoftware#87]

**Comment by jimfellows-NOAA** [comment]
I think this is related to what we talked about last week @johnharms-NOAA.  Still digging / understanding how this works, but this ticket seems to suggest that the siteID isn't currently included in the information on a fish tag or barcode.

**Comment by johnharms-NOAA** [comment]
Yes, I think it is the same issue. I guess I didn't know that this had already been added to GitHub.  Up until we "rediscovered" the problem in the lab last week, I think I always thought that setID was embedded in the barcode so that HookLogger could discern among the same ADH number from different sites.
